x/dsmr/p2p.go

```diff
@@ -38,13 +35,9 @@ var (
 		Code:    3,
 		Message: "invalid chunk",
 	}
-
-	_ acp118.Verifier = (*ChunkSignatureRequestVerifier[Tx])(nil)
-	_ p2p.Handler     = (*GetChunkHandler[Tx])(nil)
-	_ p2p.Handler     = (*ChunkCertificateGossipHandler[Tx])(nil)
```

Comment:
I don't think we have a good type to parameterize `Tx` with here to allow for these assertions... So I removed them. (Similar comment for other interface checks)